user-guide/early-access.md

```diff
@@ -6,17 +6,17 @@ Looking for early access to the Ambassador Edge Stack? [Click here to get starte
 
 From time to time, the Ambassador Edge Stack may ship early access releases to test major changes. **Early access releases are not supported for production use** but are intended to gain early feedback from our community before shipping a release.
 
-Early access releases will always have names that include the string "-ea" followed by a build number. For example, `0.50.0-ea1` is the first early access build of the Ambassador Edge Stack 0.50.0.
+Early access releases will always have names that include the string "-ea" followed by a build number.
 
 ## Early Access Status
 
-There are currently no early access releases available.
+The current early access build for the Ambassador Edge Stack is `1.0.0-ea1`.
 
 ### Installing the Ambassador Edge Stack Early Access releases
 
-Use a Kubernetes deployment as usual, but use the image `quay.io/datawire/ambassador:0.50.0-ea5`.
+Use a Kubernetes deployment as usual, but use the image `quay.io/datawire/aes:1.00.0-ea1`.
```

Comment:
Like above, you should write this as `$version$`, but also there's an extra `0` in there.

Reply:
same comments as above.

Reply:
fixed